ui/tract-renderer/src/main/java/org/cru/godtools/tract/util/LottieComposition.kt

```diff
@@ -0,0 +1,5 @@
+package org.cru.godtools.tract.util
+
+import com.airbnb.lottie.LottieComposition
+
+fun LottieComposition.scaleTo(width: Float, height: Float) = minOf(width / bounds.width(), height / bounds.height())
```

Comment:
So are you adding a method to ```LottieComposition``` or tapping into an existing method and overriding it?

Reply:
I'm defining an extension method. It's Kotlin syntactic sugar that can "add" functionality onto an existing class.

https://kotlinlang.org/docs/reference/extensions.html

In reality it gets compiled as a static method that would look like this:
```
fun scaleTo(comp: LottieComposition, width: Float, height: Float)
```